apix-integrationtests/build.gradle

```diff
@@ -13,12 +13,12 @@ subprojects {
     sourceSets {
         main {
             java {
-                srcDirs = ['src/integration-test/java-shared', 'src/main/java']
+                srcDir '../../apix-impl/src/integration-test'
```

Comment:
@WimCrols Same here as my previous comments

Reply:
Well.. I changed it for consistency, but of courese if we change it in one spot, we should do all.